lib/drab/commander.ex

```diff
@@ -240,6 +262,92 @@ defmodule Drab.Commander do
   Drab injects function `render_to_string/2` into your Commander. It is a shorthand for
   `Phoenix.View.render_to_string/3` - Drab automatically chooses the current View.
 
+  ## PubSub support
+
+  You can use the PubSub mechanism to automatically update your pages on data changes.
+
+  Suppose for example you have a backend module that interfaces the database, and you 
+  whishes to automatically update your page every time a change occours in a 
+  database table by some operations made in other part of your application. You can
+  implement
+
+  Suppose you need to automatically update your page every time a change occours in a 
+  database table by some operations made in other part of your application. You can
+  implement a PubSub mechanism to react from the commander to any events you have 
+  subscribed to: add the subscription to the desidered `PubSub` in the 
+  `onload_init` event handler, then add a `handle_info_message/2` handler for any
+  event you need to react to.
+
+  Please note that the `handle_info_message/2` handler is a simplified version of the
+  GenServer/PubSub `handle_info/2` handler, it expects a message parameter, identical 
+  to the standard `handle_info/2` one, and a `socket` parameter, instead of the
+  `state` parameters, and it doesn't have to return anything.
+
+  For example:
+
+      # the backend:
+      defmodule MyApp.Backend do
+        # PubSub stuff
+          @topic inspect(__MODULE__)
+
+          def subscribe() do
+            Phoenix.PubSub.subscribe(MyApp.PubSub, @topic)
```

Comment:
This isn't taking a PID, that's very weird...

Reply:
It is not possible to pass the pid to a PubSub.subscription function. Otherwise we could just call `Phoenix.PubSub.subscribe(MyApp.PubSub, Drab.pid(socket), @topic)` in the `onload` handler, without any need of the added sync call.

This because, although documented, `Phoenix.PubSub.subscribe/3` is deprecated. If used it works for now, but warnings with: "Passing a Pid to Phoenix.PubSub.subscribe is deprecated. Only the calling process may subscribe to topics" so we cannot anymore use it.

This is the exact reason because I've elaborate a solution through the `onload_init` stuff.

Reply:
> It is not possible to pass the pid to a PubSub.subscription function.

https://hexdocs.pm/phoenix_pubsub/Phoenix.PubSub.html#subscribe/3
Why not?  It's not marked as deprecated in the docs.

Besides, Drab already has a method to subscribe to pubsub topics via it's own `subscribe` function, it just needs to detect drab specific things and 'other' things then route the others back to the commands instead.

Reply:
https://github.com/phoenixframework/phoenix_pubsub/blob/v1.1.2/lib/phoenix/pubsub.ex#L154